pkg/ide/jetbrains.go

```diff
@@ -68,9 +68,9 @@ func downloadJetbrainsIDE(projectHostname, downloadUrl, downloadPath string) err
 
 	view_util.RenderInfoMessage(fmt.Sprintf("Downloading the IDE into the project from %s...", downloadUrl))
 
-	downloadIdeCmd := exec.Command("ssh", projectHostname, fmt.Sprintf("mkdir -p %s && curl -fsSL %s | tar -xz -C %s --strip-components=1", downloadPath, downloadUrl, downloadPath))
-	downloadIdeCmd.Stdout = io.Writer(&util.DebugLogWriter{})
-	downloadIdeCmd.Stderr = io.Writer(&util.DebugLogWriter{})
+	downloadIdeCmd := exec.Command("ssh", projectHostname, fmt.Sprintf("mkdir -p %s && wget -q --show-progress --progress=bar:force -pO- %s | tar -xzC %s --strip-components=1", downloadPath, downloadUrl, downloadPath))
```

Comment:
I'm just a passerby, but `wget` isn't installed by default on MacOS, if it matters

Reply:
This command is executed inside the project which lives in a container where wget is available. Thanks for your input nonetheless!